src/hub.js

```diff
@@ -521,22 +539,24 @@ const onReady = async () => {
   });
   environmentRoot.appendChild(initialEnvironmentEl);
 
-  const setRoom = (hubId, hubName) => {
-    if (!isBotMode) {
-      remountUI({ hubId, hubName });
+  const enterSceneWhenReady = hubId => {
+    const enterSceneImmediately = () => enterScene(new MediaStream(), false, hubId);
+    if (scene.hasLoaded) {
+      enterSceneImmediately();
     } else {
-      const enterSceneImmediately = () => enterScene(new MediaStream(), false, hubId);
-      if (scene.hasLoaded) {
-        enterSceneImmediately();
-      } else {
-        scene.addEventListener("loaded", enterSceneImmediately);
-      }
+      scene.addEventListener("loaded", enterSceneImmediately);
     }
   };
 
+  const setRoom = (hubId, hubName) => {
+    remountUI({ hubId, hubName });
+  };
+
   if (qs.has("room")) {
     // If ?room is set, this is `yarn start`, so just use a default environment and query string room.
-    setRoom(qs.get("room") || "default");
+    const hubId = qs.get("room") || "default";
+    setRoom(hubId, hubId);
+    if (isBotMode) enterSceneWhenReady();
```

Comment:
So this isn't going to work anymore if data is going over Phoenix channels, right? We should throw out this feature or fix it.

Reply:
on the fence:

- removing it seems sane to me if its any effort to keep
- it seems easy to keep: we should just have a rule to use data channels for the relevant pipes iff there is no `RETICULUM_SERVER`

Reply:
actually nevermind that is wrong. if there is no `RETICULUM_SERVER` we assume those URLs are just going to be relative :/ i'll kill this feature for now until we feel some pain. thankfully we have dev set up properly now!